MobileCenterUpdates/MobileCenterUpdates/MSUpdates.m

```diff
@@ -85,14 +107,39 @@ + (instancetype)sharedInstance {
 - (void)startWithLogManager:(id<MSLogManager>)logManager appSecret:(NSString *)appSecret {
   [super startWithLogManager:logManager appSecret:appSecret];
 
+  // TODO: Hook up with pipeline.
```

Comment:
This will be done my @jaelim-ms. I'll also take it out of `startWithLogManager`. Let's leave it for now.